learn-resources/README.markdown

```diff
@@ -70,9 +70,31 @@ That's how you link to Liferay Learn resources!
 >
 > Note: The cache refreshes every four hours by default, per the [`learn.resources.refresh.time` portal property](../portal-impl/src/portal.properties).
 
+## Previewing Liferay Learn Resource Links
+
+If you want to test your link, you don't have to recompile your module. From this folder (`learn-resources`), you can run a quick dev server that's configured with only one property/environment variable: 
```

Comment:
@sez11a could we refer to this as a `portal property` instead of just as a `property` to make it a little clearer? It seems like we have quite a few different types of [properties in Liferay](https://learn.liferay.com/search?q=property).